spec/features/post_show_spec.rb

```diff
@@ -0,0 +1,35 @@
+require 'rails_helper'
+
+RSpec.describe 'Post/show', type: :system do
+  before do
+    driven_by(:rack_test)
+    @user = User.first
+    @post = @user.posts.first
+    @comment = @post.comments.first
+  end
+
+  it 'It should shows a post\'s title' do
+    visit user_posts_path(@user, @post)
+    expect(page).to have_content(@post.title)
+  end
+
+  it 'It should shows the author of the post' do
+    visit user_posts_path(@user, @post)
+    expect(page).to have_content(@user.name)
+  end
+
+  it 'It should shows the body of the post' do
+    visit user_posts_path(@user, @post)
+    expect(page).to have_content(@post.text)
+  end
+
+  it 'It should describe the user name of each comment' do
+    visit user_posts_path(@user, @post)
+    expect(page).to have_content(@comment.user.name)
+  end
+
+  it 'It shows the comment each comment author left' do
+    visit user_posts_path(@user, @post)
+    expect(page).to have_content(@comment.text)
+  end
```

Comment:
Please add the following tests because you forgot to add them:
- [ ] a Test that asserts you can see how many comments it has as required [HERE](https://github.com/microverseinc/curriculum-rails/blob/main/blog-app/projects/intergration_spec_and_n_1_project.md#:~:text=I%20can%20see%20how%20many%20comments%20it%20has.).
- [ ] a Test that asserts you can see how many likes it has as required [HERE](https://github.com/microverseinc/curriculum-rails/blob/main/blog-app/projects/intergration_spec_and_n_1_project.md#:~:text=I%20can%20see%20how%20many%20likes%20it%20has.).